src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,120 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const {
+    gap: containerGap,
+    elevation: containerElevation,
+    round: containerRound,
+  } = theme.notification.container;
+
+  const { pad: iconContainerPad, round: iconContainerRound } =
+    theme.notification.iconContainer;
+
+  const { pad: textContainerPad } = theme.notification.textContainer;
+
+  const { size: messageTextSize, weight: messageTextWeight } =
+    theme.notification.messageText;
+
+  const { size: bodyTextSize } = theme.notification.bodyText;
+
+  const { margin: buttonMargin } = theme.notification.button;
+
+  const Icons = {
+    StatusGood: theme.notification.icons.statusGood,
+    StatusWarning: theme.notification.icons.statusWarning,
+    StatusUnknown: theme.notification.icons.statusUnknown,
+    FormClose: theme.notification.icons.closeButton,
+  };
+
+  const iconColor = theme.notification.icons.color;
+
+  let icon;
+  let color;
+  switch (status) {
+    case 'warning':
+      color = theme.notification.status.warning.color;
+      icon = <Icons.StatusWarning color={iconColor} />;
+      break;
+    case 'good':
+      color = theme.notification.status.good.color;
+      icon = <Icons.StatusGood color={iconColor} />;
+      break;
+    case 'unknown':
+      color = theme.notification.status.unknown.color;
+      icon = <Icons.StatusUnknown color={iconColor} />;
+      break;
+    default:
+      color = theme.notification.status.unknown.color;
+      icon = <Icons.StatusUnknown color={iconColor} />;
+      break;
+  }
+
+  let content = (
+    <>
+      <Box
+        round={iconContainerRound}
+        background={color}
+        pad={iconContainerPad}
+        justify="center"
+      >
+        {icon}
+      </Box>
+      <Box
+        align="start"
+        direction="row"
+        justify="between"
+        pad={textContainerPad}
+        fill
```

Comment:
I don't think we need `fill` here. Instead, `flex="grow"` might make sense.

Reply:
this was causing an issue when switching to flex="grow" in the case of having body text, see attached screenshot
<img width="535" alt="Screen Shot 2021-07-28 at 3 59 03 PM" src="https://user-images.githubusercontent.com/38971752/127387679-e93448f3-5324-4726-b620-48f3d14257cb.png">

Reply:
Maybe add a flex={false} to the box with the status icon

Reply:
<img width="525" alt="Screen Shot 2021-07-29 at 12 43 00 PM" src="https://user-images.githubusercontent.com/38971752/127531708-6a6bc8e7-6318-41b2-8e5d-cc46e72cbf88.png">

seeing this when i implement that, is there an issue with using the `fill` property? just curious if it's not best practice or should be avoided

Reply:
noticing the close icon is outside the notification here